metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -105,6 +107,21 @@ object UserConfiguration {
         |`src/main/scala:: src/main/java::`. Syntax such as `src/{main,test}::`
         |is not supported.
         |""".stripMargin
+    ),
+    UserConfigurationOption(
+      "bloop-sbt-already-installed",
+      "false",
+      "false",
+      "Don't generate Bloop plugin file for Sbt",
```

Comment:
For better or worse, the correct capitalization for sbt is always lowercase, even when starting new sentences.
```suggestion
      "Don't generate Bloop plugin file for sbt",
```